packages/app/src/settings/actions/toggle-update-available.action.test.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { AnyAction } from "@reduxjs/toolkit"
+import thunk from "redux-thunk"
+import createMockStore from "redux-mock-store"
+import { toggleUpdateAvailable } from "./toggle-update-available.action"
+
+jest.mock("App/modals-manager/actions", () => ({
+  checkAppUpdateFlowToShow: () => jest.fn(),
+}))
+
+const mockStore = createMockStore([thunk])()
+
+afterEach(() => {
+  jest.clearAllMocks()
+})
+
+test("calls `toggleTethering` action dispatch SettingsEvent.ToggleUpdateAvailable", async () => {
```

Comment:
Seems like the test description is invalid - it says something about `toggleTethering`

Reply:
https://github.com/mudita/mudita-center/pull/842/commits/706d90599383cfd7803a08c6fac03dd0e2826dc1